portfolio/src/main/webapp/script.js

```diff
@@ -147,54 +147,57 @@ function dark() {
     document.getElementById('poem_3').style.backgroundColor = 'black';
 }
 
-var amplitude = 70;
-var left_pos = 600;
-var top_pos = -600;
-var pi = Math.PI;
-var inc = pi / 30;
-var params = new Map();
-params.set("one", 0).set("two", 0).set("three", 0).set("four", 0).set("five", 0);
-params.set("six", 0).set("seven", 0).set("eight", 0).set("nine", 0).set("ten", 0);
+let amplitude = 70;
+let left_pos = 600;
+let top_pos = -600;
+let pi = Math.PI;
+let inc = pi / 30;
+let facts_animation = new Map();
 
+let facts = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"];
 
-function fly(left_pos, top_pos, position, f_ratio, s_ratio, str) {
-    var fact = document.getElementById(str);
-    var button = document.getElementById("fact");
-    var play_button = document.getElementById("bubbles");
+facts.forEach(function(item) {
+    facts_animation.set(item, "0");
+});
+
+function fly(left_pos, top_pos, position, f_ratio, s_ratio, fact_id) {
+    let fact = document.getElementById(fact_id);
```

Comment:
I'd declare and define the variables where they are used, not necessarily at the top of the scope.

This makes it easier to read the code the same way as a recipe:

let button
button...
button...

let fact
fact...
fact...

See https://google.github.io/styleguide/jsguide.html#features-declared-when-needed